lib/morse_translator.rb

```diff
@@ -38,6 +38,41 @@ def initialize
                     "9" => "----.",
                     "0" => "-----",
                     " " => " "}
+    @morse_string = ""
+
+  end
+
+
+  def compare_units(element, letter)
+    if element == letter
+      @morse_string += @dictionary[letter]
+    end
+  end
+
+
+  def process_units(array_in)
```

Comment:
This method seems to be doing quite a bit!  I start to sense some code smell anytime I see .each used multiple times in a single method!